include/rmm/mr/host/new_delete_resource.hpp

```diff
@@ -31,12 +31,12 @@ namespace mr {
  *---------------------------------------------------------------------------**/
 class new_delete_resource final : public host_memory_resource {
  public:
-  new_delete_resource()                            = default;
-  ~new_delete_resource()                           = default;
-  new_delete_resource(new_delete_resource const &) = default;
-  new_delete_resource(new_delete_resource &&)      = default;
-  new_delete_resource &operator=(new_delete_resource const &) = default;
-  new_delete_resource &operator=(new_delete_resource &&) = default;
+  new_delete_resource()                           = default;
+  ~new_delete_resource()                          = default;
+  new_delete_resource(new_delete_resource const&) = default;
+  new_delete_resource(new_delete_resource&&)      = default;
+  new_delete_resource& operator=(new_delete_resource const&) = default;
+  new_delete_resource& operator=(new_delete_resource&&) = default;
```

Comment:
I don't prefer it one way or the other, but I would have expected this to match the variable assignment rules. :shrug: 

```suggestion
  new_delete_resource()                                      = default;
  ~new_delete_resource()                                     = default;
  new_delete_resource(new_delete_resource const&)            = default;
  new_delete_resource(new_delete_resource&&)                 = default;
  new_delete_resource& operator=(new_delete_resource const&) = default;
  new_delete_resource& operator=(new_delete_resource&&)      = default;
```

Reply:
This was a fix. `operator=` previously didn't align properly.